src/config-loader.js

```diff
@@ -135,12 +135,18 @@ class ConfigLoader {
     const { include } = this.getProjectConfig();
     const {
       dir = false,
-      isEnv = false
+      isEnv = false,
+      isAllEnv = false,
```

Comment:
@KanoniPan instead of adding `isAllEnv` option i guess it would be better to merge `isEnv` and `isAllEnv` into one option `env` with values `default` (by default), `specific` and `every`